src/utils.jl

```diff
@@ -76,6 +76,9 @@ Base.getindex(D::ColVecs, i::CartesianIndex{1}) = view(D.X, :, i)
 Base.getindex(D::ColVecs, i) = ColVecs(view(D.X, :, i))
 Base.setindex!(D::ColVecs, v::AbstractVector, i) = setindex!(D.X, v, :, i)
 
+Base.vcat(a::ColVecs, b::ColVecs) = ColVecs(vcat(a.X, b.X))
```

Comment:
I guess this should be
```suggestion
Base.vcat(a::ColVecs, b::ColVecs) = ColVecs(hcat(a.X, b.X))
```